cmd/collaborators/collaborators_test.go

```diff
@@ -132,3 +134,30 @@ func TestCollaboratorsCommand(t *testing.T) {
 		})
 	}
 }
+
+func TestCollaboratorsCommand_PrintSuccess(t *testing.T) {
```

Comment:
thought: I'm a little baffled where this test came from 🤔 I didn't write it, so I assume I updated it. But I cannot find the original copy of it! Either way, it tests `printSuccess` and I think we want to keep it. I'm assuming it was introduced by a rebase (kinda regretting doing a rebase instead of a merge).

Reply:
Ohhhh, great memory! I'll remove this test then!